csharp/ql/test/library-tests/assignables/Assignables.expected

```diff
@@ -39,39 +39,12 @@
 | Assignables.cs:78:22:78:22 | s |
 | Assignables.cs:82:21:82:24 | temp |
 | Assignables.cs:84:30:84:30 | e |
-| Assignables.cs:92:10:92:14 | Item1 |
```

Comment:
These have been removed because they are not `fromSource()`. Does this cause any issue?

Reply:
I think that is fine.